Demo/Common/IntegrationExamples/DropIn.swift

```diff
@@ -19,10 +19,8 @@ internal final class DropIn: DropInExampleProtocol {
 
     internal var dropInComponent: PresentableComponent?
     internal weak var presenter: PresenterExampleProtocol?
-    internal var paymentMethods: PaymentMethods?
+    //internal var paymentMethods: PaymentMethods?
```

Comment:
Could be removed :)

Reply:
yeah 😄 . removed d5b5eb7f2d00b5eaf1196654aeeda1cddc1beaea